src/common/com/intellij/plugins/haxe/model/HaxeClassModel.java

```diff
@@ -72,6 +111,33 @@ public HaxeClassModel getParentClass() {
     return out;
   }
 
+  public HaxeClassModel getAliasOrSelf() {
+    final ResultHolder type = getAliasType();
+    final SpecificHaxeClassReference type1 = type != null ? type.getClassType() : null;
+    if (type1 != null) {
+      return type1.getHaxeClassModel();
+    }
+    return this;
+  }
+
+  @Nullable
+  public ResultHolder getAliasType() {
```

Comment:
A typedef could be more than an alias. A typedef could be an anonymous structure too.

https://github.com/Akamon/intellij-haxe/commit/033ad0e42446333eb858c7c77a24fa373b071efe

So to not be confused by alias, but to not be that generic, I'm using AliasTypedef :)